src/__tests__/commonjs-import/webpack.config.js

```diff
@@ -1,6 +1,6 @@
 const baseConfig = require('../__utils__/webpack.config')(__dirname);
 
-const { InlineRequireWebpackPlugin } = require('../..');
+const { InlineRequireWebpackPlugin } = require('../../../build');
```

Comment:
Had to switch tests to build so that dedicated workers could be spawned from the worker file (wouldn't work with TS).
Makes tests a little slower, but also has the added benefit of ensuring that code that'd be used by users pass sanity.